browser/main/lib/MenuManager.js

```diff
@@ -0,0 +1,35 @@
+const OSX = global.process.platform === 'darwin'
```

Comment:
Yes. I'll create `KeysManager` and transplant Hotkeys into it and will delete `MenuManager` since the day when I really need it.